controllers/toolchainclusterhealth/healthchecker_test.go

```diff
@@ -0,0 +1,176 @@
+package toolchainclusterhealth
+
+import (
+	"context"
+	"testing"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/codeready-toolchain/toolchain-common/pkg/cluster"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/h2non/gock.v1"
+	corev1 "k8s.io/api/core/v1"
+	kubeclientset "k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var logger = logf.Log.WithName("toolchaincluster_healthcheck")
+
+func TestClusterHealthChecks(t *testing.T) {
+	// given
+	defer gock.Off()
+	tcNs := "test-namespace"
+	gock.New("http://cluster.com").
```

Comment:
as these tests lives in the same package the code they are testing is, I'd say it is preferable to have this logger inside the test function

```suggestion
func TestClusterHealthChecks(t *testing.T) {
	// given
	logger := logf.Log.WithName("toolchaincluster_healthcheck_test")
	defer gock.Off()
	tcNs := "test-namespace"
	gock.New("http://cluster.com").
```